snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/rest/io/SnomedImportApiTest.java

```diff
@@ -203,5 +226,41 @@ public void import11ExtensionConceptWithVersion() {
 		getComponent(branchPath, SnomedComponentType.CONCEPT, "555231000005107").statusCode(200);
 		getVersion("SNOMEDCT-NE", "2015-02-05").statusCode(200);
 	}
+	
+	@Test
+	public void import12BranchHeadTimestampUpdateAfterImporting() {
+		final Branching branches = RepositoryRequests.branching();
+		
+		final String repositoryId = SnomedDatastoreActivator.REPOSITORY_UUID;
+		
+		final String branchA = "a";
+		
+		final Branch branchBeforeImport = branches.prepareCreate()
+				.setParent(Branch.MAIN_PATH)
+				.setName(branchA)
+				.build(repositoryId)
+				.execute(getBus())
+				.getSync(1, TimeUnit.MINUTES);
+		
+		assertEquals("After branch creation timestamps should still be equal", branchBeforeImport.baseTimestamp(), branchBeforeImport.headTimestamp());
+		
+		importArchive("SnomedCT_Release_INT_20160131_unique_concept.zip", branchBeforeImport.path(), false, Rf2ReleaseType.DELTA.name());
+	
+		final Branch branchAfterImport = branches.prepareGet(branchBeforeImport.path())
+				.build(repositoryId)
+				.execute(getBus())
+				.getSync(1, TimeUnit.MINUTES);
+		
+		IBranchPath path = BranchPathUtils.createPath(branchAfterImport.path());
+		
+		getComponent(path, SnomedComponentType.CONCEPT, "301795004").statusCode(200);
+		
+		
+		assertThat(branchBeforeImport.baseTimestamp(), not(equalTo(branchAfterImport.headTimestamp())));
```

Comment:
Use a shorter version of assertion to increase readability: `assertNotEquals(branchAfterImport.baseTimestamp(), branchAfterImport.headTimestamp())`.